testdata/project-v4-declarative-v1/monitoring/tools/metricsdocs.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+
+	"sigs.k8s.io/kubebuilder/testdata/project-v4-declarative-v1/monitoring/metrics"
+	"sigs.k8s.io/kubebuilder/testdata/project-v4-declarative-v1/monitoring/metrics/util"
+)
+
+// constant parts of the file
+const (
+	title      = "# sigs.k8s.io/kubebuilder/testdata/project-v4-declarative-v1 Operator Metrics\n"
+	background = "This document aims to help users that are not familiar with metrics exposed by the sigs.k8s.io/kubebuilder/testdata/project-v4-declarative-v1 operator.\n" +
```

Comment:
we cannot scaffold it by default. 
The final result of the plugin should not point out users for the master samples.

Reply:
Also, same of : https://github.com/kubernetes-sigs/kubebuilder/pull/3106/files#r1037114035 

It does not fit to an end-project scaffold. It should be a lib, tool and etc.

Reply:
The plan is go give and example scaffold for adding new metrics, the files can be commented also.
but it's not by default, the user will need to init/edit with monitoring plugin in order to get these files.